bootstrap.sh

```diff
@@ -43,8 +43,15 @@ if [ -z $python ]; then
     python=python3
 fi
 
-pyver=$($python -V | sed -n 's/Python \([0-9]\+\)\.\([0-9]\+\)\..*/\1.\2/p')
+while [ -n "$1" ]; do
+    case "$1" in
+        "+docs") MAKEDOCS="true" && shift ;;
+        "+pygelf") PYGELF="true" && shift ;;
+        *) usage && exit 0 ;;
```

Comment:
This is an error, you should exit with non-zero:

```suggestion
        *) usage && exit 1 ;;
```